package.json

```diff
@@ -22,7 +22,8 @@
     "build:clean": "npm run clean && npm run initdev && npm run build",
     "build:tasks": "npm run compile:tasks",
     "package": "npm run build:clean && npm run package:tasks",
-    "clean": "git clean -fdX"
+    "clean": "git clean -fdX",
+    "patchvspublish": "node ./scripts/patchTaskAsDev --task=PublishVSExtension --patch"
```

Comment:
Can you pull this out of the pull request? This is what the "Update task Ids" feature of the tasks are for.

Reply:
This is for me to test my individual task by directly uploading it from my machine. It helps me to see the task in the pipeline before commit.